src/components/UnknownTokensListItem.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright 2018 - 2022 The Alephium Authors
+This file is part of the alephium project.
+
+The library is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+The library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
```

Comment:
<img width="381" alt="image" src="https://github.com/alephium/mobile-wallet/assets/6715943/b5858c9d-9613-4780-8dae-5f49a78d4f08">

A note for our future us on this component's design: let's not use a ListItem as it may be confusing when quickly looking at the list IMO. 

I'll redesign it late. Could you add a little todo in the code to make sure I don't miss it? Thanks!

Reply:
TODO added!